extensions/ql-vscode/src/view/model-editor/__tests__/ModelEvaluation.spec.tsx

```diff
@@ -0,0 +1,127 @@
+import { render as reactRender, screen } from "@testing-library/react";
+import type { Props } from "../ModelEvaluation";
+import { ModelEvaluation } from "../ModelEvaluation";
+import { createMockModelEditorViewState } from "../../../../test/factories/model-editor/view-state";
+import type { ModeledMethod } from "../../../model-editor/modeled-method";
+import { createMethod } from "../../../../test/factories/model-editor/method-factories";
+import { createMockVariantAnalysis } from "../../../../test/factories/variant-analysis/shared/variant-analysis";
+import { VariantAnalysisStatus } from "../../../variant-analysis/shared/variant-analysis";
+
+describe(ModelEvaluation.name, () => {
+  const method = createMethod();
+  const modeledMethod: ModeledMethod = {
+    ...method,
+    type: "summary",
+    input: "Argument[0]",
+    output: "ReturnValue",
+    kind: "taint",
+    provenance: "manual",
+  };
+  const modeledMethodsMap: Record<string, ModeledMethod[]> = {};
+  modeledMethodsMap[method.signature] = [modeledMethod];
```

Comment:
You could use `createSummaryModeledMethod(method)` to create a modeled method that almost identical to the one you're making here.

That then makes it a bit nicer to make this code more compact, like:
```suggestion
  const modeledMethodsMap: Record<string, ModeledMethod[]> = {};
  modeledMethodsMap[method.signature] = [createSummaryModeledMethod(method)];
```

Reply:
Ooh, that's perfect, thank you! ⚡